examples/simple-average/run.py

```diff
@@ -15,31 +14,27 @@
   tfe.set_config(config)
   tfe.set_protocol(tfe.protocol.Pond())
 
-
+@tfe.local_computation
 def provide_input() -> tf.Tensor:
   # pick random tensor to be averaged
   return tf.random_normal(shape=(10,))
 
+@tfe.local_computation('result-receiver')
+def receive_output(average: tf.Tensor) -> tf.Operation:
+  # simply print average
+  return tf.print("Average:", average)
+
 
 if __name__ == '__main__':
   # get input from inputters as private values
-  inputs = [
-      tfe.define_private_input('inputter-0', provide_input),
-      tfe.define_private_input('inputter-1', provide_input),
-      tfe.define_private_input('inputter-2', provide_input),
-      tfe.define_private_input('inputter-3', provide_input),
-      tfe.define_private_input('inputter-4', provide_input),
-  ]
+  inputs = [provide_input(player_name="inputter-{}".format(i))  # pylint: disable=unexpected-keyword-arg
```

Comment:
fyi the explicit list was intentional to make it super clear that we're dealing with five inputters